src/wam2layers/preprocessing/era5.py

```diff
@@ -205,15 +260,15 @@ def get_dp_pressurelevels(q, u, v, ps, qs, us, vs):
     assert np.all(dp >= 0), "Pressure levels should increase monotonically"
 
     # Interpolate to midpoints
-    midpoints = 0.5 * (u.level.values[1:] + u.level.values[:-1])
+    midpoints = 0.5 * (u["level"].to_numpy()[1:] + u["level"].to_numpy()[:-1])
```

Comment:
What's the difference between `.values` and `to_numpy`?

Reply:
Posted this above already, but `.to_numpy()` does the same as `.values` but is more explicit. In pandas [they are already asking](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.values.html) for users to move to this syntax, in xarray not yet.

Reply:
Thanks, didn't know that!